bridge/README.md

```diff
@@ -53,11 +50,52 @@ docker run -it \
     quay.io/domino/bridge:latest init sagemaker
 ```
 
-#### 2. Run Bridge
+### 2. Run Bridge
+
+#### AWS Config
+
+After you have initialized Bridge once, you can switch to AWS
+credentials with more restrictive access. The AWS CLI in your
+shell needs to have access to:
+
+- List/describe/create/delete SageMaker Endpoints, EndpointConfigs, and Models
```

Comment:
If we are going to propose this, can we provide a policy document for them to copy/paste?